scripts/variantstore/wdl/GvsPrepareCallset.wdl

```diff
@@ -70,8 +74,14 @@ task PrepareCallsetTask {
             --query_project ~{query_project} \
             --fq_sample_mapping_table ~{fq_sample_mapping_table} \
             ~{"--sa_key_path " + service_account_json}
+
+      echo ~{fq_destination_dataset}.~{destination_cohort_table_name} > fq_cohort_extract_table.txt
     >>>
 
+    output {
+      String fq_cohort_extract_table = read_string("fq_cohort_extract_table.txt")
```

Comment:
My WDL is rusty, but do you have to go through a file here?  Can you just do something like

```
String fq_cohort_extract_table = "~{fq_destination_dataset}.~{destination_cohort_table_name}"
```

or something similar?

Reply:
that works! thanks for the suggestion

Reply:
actually, ended up backtracking on this because I added some more bash interpolation logic to generate the output string. unless you know of a way to directly map a bash env variable -> cromwell output?